<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1850</width>
    <height>950</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>1850</width>
      <height>891</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="InButton">
    <property name="geometry">
     <rect>
      <x>1650</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 255, 143);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>放大 +</string>
    </property>
   </widget>
   <widget class="QPushButton" name="outButton">
    <property name="geometry">
     <rect>
      <x>1750</x>
      <y>0</y>
      <width>93</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(160, 255, 143);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>缩小 -</string>
    </property>
   </widget>
   <widget class="QComboBox" name="startBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string> 起始点：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 75 10pt &quot;Consolas&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string> 终点：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="endBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 127);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>最短距离为：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shortBtn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>开始查找最短路径</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="showDist">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>0</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 10pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>   </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1850</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
